aries-site/src/utils/hooks/usePersistedState.js

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect, useRef } from 'react';
+import { createGlobalState, useEventListener } from '..';
+
```

Comment:
It would be helpful to add comments to this file to explain purpose / why it exists (in this case, the file name is helpful, but an elaborated description would help).

Reply:
Agreed.